scripts/README.md

```diff
@@ -41,3 +41,68 @@ Usage:
    $ ./get-image-changes.sh VERSION-TO-UPDATE > changes
    $ cat changes
 ```
+
+## `kernel_monitor/kernel_monitor.py`
+
+This script fetches the latest kernel information and looks for new LTS releases.
+If it finds a new LTS release it sends an email to the selected recipients.
+
+Requirements:
+
+* python3
+* python dependencies (see the [requirements file](https://github.com/clearcontainers/packaging/scripts/kernel_monitor/requirements))
+* pip
+* postfix
+
+Usage:
+
+NOTE: The following commands can be executed if you have a Fedora 27 system that meets all requirements,
+and a postfix server running on 127.0.0.1 or localhost.
+
+Install python dependencies:
+
+NOTE: It is strongly recommended to run this script in a Python virtual environment with `virtualenv`.
+```
+$ cd kernel_monitor
+$ virtualenv km
+$ source km/bin/activate
+$ pip install -r requirements
+```
+
+Set distro_certs variable to: `/etc/ssl/certs/ca-bundle.crt`:
+
+NOTE: This path might change based on the Linux distribution you are using (see kernel_monitor.py).
+```
+sed -i "s/distro_certs = .*/distro_certs = \"\/etc\/ssl\/certs\/ca-bundle.crt\"/" kernel_monitor.py
```

Comment:
Couple of points:

- This line would need a `$ ` prompt.
- I'm really not sure we want users to modify the script itself. Could you add an option for specifying the path to the certs file to avoid this?